src/utils/ParseRouteUtils.js

```diff
@@ -150,11 +150,14 @@ async function condenseRoute(
      * Discard routes where not possible to walk to bus given departure buffer
      * We subtract (ONE_MIN_IN_MS - 1) from departureTimeNow to avoid the
      * case where we don't show a bus leaving at the same minute as departureTime.
+     * We also have to account for the delay time because if a bus is delayed, we want to display
+     * the route even if the start time is earlier than the current time.
      */
-    if (departureDelayBuffer
-      && direction.type === 'depart'
-      && startTime < departureTimeNowMs - (ONE_MIN_IN_MS - 1)) {
-      return null;
+    if (departureDelayBuffer && direction.type === 'depart') {
+      const delayMs = direction.delay !== null ? direction.delay * 1000 : 0;
+      if (startTime < departureTimeNowMs - (ONE_MIN_IN_MS - 1) - delayMs) {
```

Comment:
Do we need to subtract `1`, since we have `<` and not `<=`? If we don't, we should drop the parentheses